sdk/feature-management-applicationinsights-node/.eslintrc

```diff
@@ -0,0 +1,62 @@
+{
+    "env": {
+        "browser": true,
```

Comment:
Should some of these be false for node?

Reply:
Interesting, I copied this file and didn't check the content. I think you are correct. Also update it for the browser package.